src/libraries/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -869,6 +1183,59 @@ public void HandleEvent(SocketAsyncContext context)
                 op.Dispatch();
             }
 
+            internal void BatchOrDispatch(SocketAsyncContext context, Span<Interop.Sys.IoControlBlock> ioControlBlocks, Span<AsyncOperation> batchedOperations, ref int batchedCount)
```

Comment:
important: there are few async socket operations that can not be batched, so we batch what we can and dispatch the rest